utils/ChromiumDownloader.js

```diff
@@ -37,6 +37,21 @@ const downloadURLs = {
   win64: '%s/chromium-browser-snapshots/Win_x64/%d/chrome-win32.zip',
 };
 
+const protocolSelector = (function() {
+  const protocols = {
+    'http:': require('http'),
+    'https:': require('https'),
```

Comment:
and http2 if available

Reply:
https is already required at [line 18](https://github.com/GoogleChrome/puppeteer/pull/1372/files#diff-86a9034561c2112f5ab45ccf6a6f8415R18)

Reply:
Remove https from line 18. As of now it's a bit low level to detect support for HTTP/2 for this we would need in my mind a library that handled it all for us, one with unit tests so that we don't have to test and maintain that part here.

This could be done, but I guess it's a little overkill for a install script that downloads one file.